draft-ietf-add-ddr.md

```diff
@@ -255,22 +255,25 @@ is a private or local address.
 
 A client MAY additionally use a discovered Designated Resolver without
 either of these methods, based on implementation-specific policy or user input.
-Details of such policy are out of scope of this document. Clients SHOULD NOT
+Details of such policy are out of scope of this document. Clients MUST NOT
 automatically use a Designated Resolver without some sort of validation,
 such as the two methods defined in this document or a future mechanism.
 
-A client MUST NOT use a Designated Resolver designated by one Unencrypted Resolver
-in place of another Unencrypted Resolver. As these are known only by IP address,
-this means each unique IP address used for unencrypted DNS requires its own
-designation discovery. This ensures queries are being sent to a party designated by
-the resolver originally being used.
+A client MUST NOT re-use a designation discovered using the IP address of one
+Unencrypted Resolver in place of any other Unencrypted Resolver. Instead, the client
+SHOULD repeat the discovery process to discover the Designated Resolver of the other
+Unencrypted Resolver. In other words, designations are per-resolver and MUST
+NOT be used to configure the client's universal DNS behavior. This ensures
+in all cases that queries are being sent to a party designated by the resolver
+originally being used. This cannot be guaranteed when designations are reused because
+Unencrypted Resolvers can only be identified by their IP address.
```

Comment:
I'm not sure this last sentence adds value, and it almost makes it sound like this is done ("when designations are reused"). We could reword to make it more hypothetical ("if you did this, it would be bad"), but I think the point is already made.

```suggestion
originally being used.
```

Reply:
I had debated this sentence in the first place but wanted to ensure every point from the original text was included. Glad to hear it's redundant.